client/dom/svg.legend.js

```diff
@@ -208,7 +208,8 @@ export default function svgLegend(opts) {
 				tickSize: 2,
 				topTicks: true
 			}
-			if (d.termid.toLowerCase().includes('cnv') && min < 0 && max > 0) {
+			// some color scales (such as brainImaging) do not have termid
+			if (d.termid?.toLowerCase().includes('cnv') && min < 0 && max > 0) {
```

Comment:
terrible fix to hardcode such logic. must improve or replace this code

Reply:
I will ask Colleen to replace it

Reply:
Colleen will fix in another PR